06-best-practices-R.md

```diff
@@ -41,11 +41,11 @@ The following error message indicates that R has failed to set the working direc
 Error in setwd("~/path/to/working/directory") : cannot change working directory
 ```
 
-Consider using the convention that the user running the script should begin in the relevant directory on their machine and then use relative file paths (see below). 
+Note that using `~` for the home directory is a convention on Unix systems and will allow others to run your code more easily on their machines than if the path specified your username.  Consider using the convention that the user running the script should begin in the relevant directory on their machine and then use relative file paths (see below).
```

Comment:
I don't think this extra sentence explaining the tilde is necessary for two reasons:

1. The prerequisite for this lesson is an understanding of how to navigate a Unix filesystem.
2. The point is to use relative paths instead of absolute paths. While using a tilde is better than specifying the name of your home directory, it would still force others to put the directory in the exact same place relative to their home directory.